src/babylon.scene.ts

```diff
@@ -1846,7 +1863,7 @@ module BABYLON {
                 if (!checkPicking && ActionManager && ActionManager.HasPickTriggers) {
                     act = this._initActionManager(act, clickInfo);
                     if (act) {
-                        checkPicking = act.hasPickTriggers;
+                        checkPicking = act.hasPickTriggers;

```

Comment:
those changes are probably Line ending change. I guess that wrong line ending have been added by the linter. (or is my config wrong ?)

Reply:
Yes but no worry (you can now always run gulp --tsLintFix)